plugins/operators/upload.py

```
@@ -0,0 +1,65 @@
+"""
+Upload files operator.
+
+| Copyright 2017-2026, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+import fiftyone.operators as foo
+import fiftyone.operators.types as types
+
+DEFAULT_DESTINATION = "/tmp"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)